scripts/bdc/get_bdc_studies_md_from_gen3.py

```diff
@@ -0,0 +1,269 @@
+                        'Description': description,
+                        'Program': '|'.join(sorted(set(filter(None, program_names)))),
+                        'Last modified': last_modified,
+                        'Notes': notes.strip()
+                    })
+                    
+                except requests.exceptions.RequestException as e:
+                    self.logger.error(f"Error processing study {study_id}: {e}")
+                    continue
+        
+        return self.raw_file
+    
+    def filter_studies(self):
+        """Filter studies and create cleaned output"""
+        self.filtered_file = os.path.join(self.output_dir, "gen3_studies_filtered.csv")
+        self.logger.info(f"Starting study filtering process. Output file: {self.filtered_file}")
+        
+        total_studies = 0
+        valid_studies = 0
+        skipped_studies = 0
+        valid_study_details = []
+        skipped_details = []
+        
+        try:
+            with open(self.raw_file, 'r', encoding='utf-8') as infile, \
+                 open(self.filtered_file, 'w', newline='', encoding='utf-8') as outfile:
+                
+                reader = csv.DictReader(infile)
+                writer = csv.DictWriter(outfile, fieldnames=reader.fieldnames)
+                writer.writeheader()
+                
+                for row in reader:
+                    total_studies += 1
+                    is_valid, invalid_reason = self.validate_study(row)
+                    
+                    if is_valid:
+                        writer.writerow(row)
+                        valid_studies += 1
+                        study_id = f"{row['Accession']}{row.get('Consent', '')}"
+                        valid_study_details.append(study_id)
+                    else:
+                        skipped_studies += 1
+                        study_id = f"{row['Accession']}{row.get('Consent', '')}"
+                        skipped_details.append(f"Study {study_id}: {invalid_reason}")
+            
+            # Write summary
+            self.write_summary(total_studies, valid_studies, skipped_studies, 
+                             valid_study_details, skipped_details)
+            
+        except Exception as e:
+            self.logger.error(f"An error occurred during filtering: {str(e)}")
+            raise
+
+    def validate_study(self, row):
+        """Validate if a study has all required fields that are required to xml file"""
+        required_fields = ['Accession', 'Consent', 'Study Name', 'Program', 'Description']
+        missing_fields = []
+        
+        for field in required_fields:
+            if not row.get(field) or row[field].strip() == '':
+                missing_fields.append(field)
+        
+        if missing_fields:
+            return False, f"Missing required fields: {', '.join(missing_fields)}"
+        return True, None
+
+    def write_summary(self, total, valid, skipped, valid_details, skipped_details):
```

Comment:
I've become fond of writing out the overall summary as a JSON document so that it's easier to parse it for downstream applications, but this is easier to read -- so it's up to you which of those criteria are more important for this task!